website/src/components/EventSlider/index.tsx

```diff
@@ -155,7 +155,7 @@ const EventSlider: React.FC<EventsProps> = ({
       height="100%"
     >
       <Typography variant="h4" className={classes.noEventText}>
-        {t("community.communityEvents.noEvent.message")}
+      <a className = {classes.noEventLink} href = "https://community.cncf.io/openebs-community">{t("community.communityEvents.noEvent.message")}</a>
```

Comment:
Use `Link` component and get the URL from constants

Reply:
Please do the same in other places as well

Reply:
done